kubernetes/samples/quick-start/quick-start-domain-resource.yaml

```diff
@@ -0,0 +1,150 @@
+  imagePullSecrets:
+  - name: weblogic-repo-credentials
+
+  # Identify which Secret contains the WebLogic Admin credentials,
+  # the secret must contain 'username' and 'password' fields.
+  webLogicCredentialsSecret:
+    name: sample-domain1-weblogic-credentials
+
+  # Whether to include the WebLogic Server stdout in the pod's stdout, default is true
+  includeServerOutInPodLog: true
+
+  # Whether to enable overriding your log file location, see also 'logHome'
+  #logHomeEnabled: false
+
+  # The location for domain log, server logs, server out, introspector out, and Node Manager log files
+  # see also 'logHomeEnabled', 'volumes', and 'volumeMounts'.
+  #logHome: /shared/logs/sample-domain1
+
+  # Set which WebLogic Servers the Operator will start
+  # - "NEVER" will not start any server in the domain
+  # - "ADMIN_ONLY" will start up only the administration server (no managed servers will be started)
+  # - "IF_NEEDED" will start all non-clustered servers, including the administration server, and clustered servers up to their replica count.
+  serverStartPolicy: "IF_NEEDED"
+
+  # Settings for all server pods in the domain including the introspector job pod
+  serverPod:
+    # Optional new or overridden environment variables for the domain's pods
+    # - This sample uses CUSTOM_DOMAIN_NAME in its image model file
+    #   to set the WebLogic domain name
+    env:
+    - name: CUSTOM_DOMAIN_NAME
+      value: "domain1"
+    - name: JAVA_OPTIONS
+      value: "-Dweblogic.StdoutDebugEnabled=false"
+    - name: USER_MEM_ARGS
+      value: "-Djava.security.egd=file:/dev/./urandom -Xms256m -Xmx512m "
+    resources:
+      requests:
+        cpu: "250m"
+        memory: "768Mi"
+
+    # Optional volumes and mounts for the domain's pods. See also 'logHome'.
+    #volumes:
+    #- name: weblogic-domain-storage-volume
+    #  persistentVolumeClaim:
+    #    claimName: sample-domain1-weblogic-sample-pvc
+    #volumeMounts:
+    #- mountPath: /shared
+    #  name: weblogic-domain-storage-volume
+
+  # The desired behavior for starting the domain's administration server.
+  adminServer:
+    # The serverStartState legal values are "RUNNING" or "ADMIN"
+    # "RUNNING" means the listed server will be started up to "RUNNING" mode
+    # "ADMIN" means the listed server will be start up to "ADMIN" mode
+    serverStartState: "RUNNING"
+    # Setup a Kubernetes node port for the administration server default channel
+    #adminService:
+    #  channels:
+    #  - channelName: default
+    #    nodePort: 30701
+
+  # The number of managed servers to start for unlisted clusters
+  replicas: 1
+
+  # The desired behavior for starting a specific cluster's member servers
+  clusters:
+  - clusterName: cluster-1
+    serverStartState: "RUNNING"
+    serverPod:
+      # Instructs Kubernetes scheduler to prefer nodes for new cluster members where there are not
+      # already members of the same cluster.
+      affinity:
```

Comment:
I removed the affinity since it simplifies the domain resource yaml and is probably not important to override the default anti-affinity behavior for quick start. Let me know if we need to add it back and comment out the lines.